chrome/locale/en-US/options.dtd

```diff
@@ -97,3 +97,8 @@
 <!ENTITY acc.boolean.value.0.label "Use default value">
 <!ENTITY acc.boolean.value.1.label "Yes">
 <!ENTITY acc.boolean.value.2.label "No">
+
+<!ENTITY policy.unsigned_header.mode "Mode to handle unsigned headers, which should be signed:">
```

Comment:
What is the reason behind changing the strings?

Reply:
Actually I was unhappy with the German version. Now I'm using the same string in other languages and nearly the same in the German version. 
I hope this fits your needs.

Reply:
Note that I am not against changing and improving the messages. Just would like to keep them in sync, so any improvements would also need to be done for master.

So just wanted to make sure this is a general string improvement you would like to suggest. And not e.g. just because the smaller option dialog in the 2.x versions needs a different (shorter) message.

Reply:
Ok, I'd like to propose to use my changes for the German version from the latest commit. I can also create a PR for master, if you agree.